src/app/find/mapPage/filters/filters.css

```diff
@@ -0,0 +1,69 @@
+@value primaryColor from "../../../../colors.css";
+
+.doneBtnContainer{
+	position: fixed;
+	bottom: 0;
+	width: 100%;
+	z-index: 1040;
```

Comment:
1040 seems pretty random. Where does this number come from?

Reply:
Dunno. What should it be?

Reply:
Haha. Well, 1 greater than whatever you want this to be in front of, I guess. :)
What's this supposed to be covering?